### ISSUE TITLE
Sometime module_eval can not be changed to define_method.

### ISSUE THREAD

**Issue by u2** [DamirSvrtan/fasterer#6]
I have play `fasterer` in my code.It point out `Using module_eval is slower than define_method. Occured at lines: 50.`.But my code is something like that:
```
class Thing
 @foo = 'test'
end
Thing.module_eval {puts @foo}
```
It can not be changed to `define_method` in this situation.So, it think it should not give me the suggestion anywhere I using `module_eval`.

**Comment by DamirSvrtan** [comment]
Hey, thnx for reporting. I will try to change after we see what happens with this issue: https://github.com/JuanitoFatas/fast-ruby/issues/22

**Comment by DamirSvrtan** [comment]
For now I've added a check to see if module_eval has a def inside of it. It's released in version 0.1.5. 
I'm following the fast-ruby repository so if that changes i will remove it from fasterer.

Thnx for reporting.
